codegen/CodeGenerator/src/main/java/ProcessFunctionWriter.java

```diff
@@ -33,4 +37,59 @@ String keyListToCode(@Nullable List<String> keyList) {
 
         return code.toString();
     }
+
+    void expressionToCode(final Expression expression, StringBuilder code) {
+        requireNonNull(code);
+        requireNonNull(expression);
+        List<Value> values = expression.getValues();
+        valueToCode(values.get(0), code);
```

Comment:
In fact, the values of an expression can be another expression.  It does not happen in the Q6 but will happen later.  Such issues do not need to be fixed in the branch.